common/mariadb/templates/deployment.yaml

```diff
@@ -53,6 +53,24 @@ spec:
                   operator: In
                   values:
                   - {{.Values.global.region}}a
+      initContainers:
+      - name: set-root-password
+        image: {{ required ".Values.global.dockerHubMirrorAlternateRegion is missing" .Values.global.dockerHubMirrorAlternateRegion }}/{{ .Values.image }}
+        imagePullPolicy: {{ default "IfNotPresent" .Values.imagePullPolicy | quote }}
+        command:
+          - /bin/bash
+          - -c
+          - |
+            /configurator/mariadb-set-root-password.sh
+        volumeMounts:
```

Comment:
Shouldn't we mount the /var/lib/mysql here too?
```
- name: mariadb-persistent-storage
            mountPath: /var/lib/mysql
            readOnly: false
```

Reply:
This init container only needs one shared with the mariadb container directory, where it puts the generated root init.sql